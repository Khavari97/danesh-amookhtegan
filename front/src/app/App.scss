@import 'assets/fonts/fonts';

* {
  margin: 0;
  font-family: "Shabnam", serif;
}

a {
  text-decoration: none;
}

//Slider css
.alice-carousel__dots {
  .alice-carousel__dots-item:not(.__custom).__active {
    background-color: #046B00;
  }
}


