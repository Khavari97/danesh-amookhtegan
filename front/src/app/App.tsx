import React from "react";
import "./App.scss";
import AppRouter from "./navigation/AppRouter";
import {SnackbarProvider} from 'notistack';
import {SnackbarUtils} from "./components/material-ui/Snackbar/SnackbarUtils";
import ThemeProvider from "./assets/styles/material/material";

export interface AppProps {
}

export interface AppState {
}

class App extends React.Component<AppProps, AppState> {
    constructor(props: AppProps) {
        super(props);
        this.state = {};
    }

    componentDidMount = () => {
    }

    render() {
        return (
            <SnackbarProvider maxSnack={3} anchorOrigin={{horizontal: 'right', vertical: 'bottom'}}>
                <SnackbarUtils/>
                <ThemeProvider>
                    <AppRouter/>
                </ThemeProvider>
            </SnackbarProvider>
        );
    }
}

export default App;
