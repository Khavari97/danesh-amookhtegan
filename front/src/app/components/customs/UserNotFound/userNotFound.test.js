import React from "react"
import Enzyme, { shallow,configure,mount } from "enzyme";
import renderer from "react-test-renderer";
import App from './UserNotFound';
import Adapter from 'enzyme-adapter-react-16'
import {Typography} from '@material-ui/core';

configure({ adapter: new Adapter() })

const setup = (props = {}) => {
  return shallow(<App {...props} />);
};

const findByTestAttr = (wrapper, val) => {
    return wrapper.find(`[data-test="${val}"]`);
};

describe('render userNotFound component without error',()=>{
    let wrapper = setup();

    it('it should have value', () => {
          expect(wrapper.find(Typography).text()).toBe('کاربری با این مشخصات یافت نشد!');
          expect(wrapper.find(Typography).prop('color')).toBe('primary');
          expect(wrapper).toBeTruthy();
          expect(wrapper.length).toEqual(1);
    });

    it('snapshot',()=>{
        const shot = <App />
        const snapshot = renderer.create(shot).toJSON();
        expect(snapshot).toMatchSnapshot();
    })
})