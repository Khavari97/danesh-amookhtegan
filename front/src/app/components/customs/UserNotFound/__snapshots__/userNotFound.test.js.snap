// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render userNotFound component without error snapshot 1`] = `
<div
  className="MuiGrid-root MuiGrid-container MuiGrid-align-items-xs-center MuiGrid-justify-xs-center"
  style={
    Object {
      "marginTop": "50px",
    }
  }
>
  <svg
    height="40"
    viewBox="0 0 65 64.608"
    width="40"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g
      data-name="Group 2"
      id="Group_2"
      transform="translate(-1371 -235)"
    >
      <path
        d="M25.006.156c-7.251,0-13.4,4.978-13.4,14.5A24.848,24.848,0,0,0,18.186,31.12c1.47,3.9-1.176,5.36-1.725,5.566C8.878,39.469,0,44.525,0,49.542v1.881c0,6.83,13.013,8.388,25.085,8.388a89.321,89.321,0,0,0,13.248-.862A16.321,16.321,0,0,1,31.513,33a9.781,9.781,0,0,1,.47-1.96H31.9A24.871,24.871,0,0,0,38.49,14.658C38.49,5.134,32.248.156,25.006.156ZM43.9,29.866a13.8,13.8,0,1,0,8.623,24.614l.549.549a2.4,2.4,0,0,0,.392,2.822l6.193,6.193a2.52,2.52,0,0,0,3.527,0l1.1-1.1a2.52,2.52,0,0,0,0-3.527l-6.193-6.193a2.438,2.438,0,0,0-2.9-.392l-.549-.549A13.769,13.769,0,0,0,43.9,29.866Zm0,3.763A10.034,10.034,0,1,1,33.864,43.663,10.028,10.028,0,0,1,43.9,33.629Z"
        fill="#046b00"
        id="icons8_find_user_male"
        transform="translate(1371 234.844)"
      />
      <g
        data-name="Group 1"
        id="Group_1"
        transform="translate(95 -73)"
      >
        <line
          data-name="Line 1"
          fill="none"
          id="Line_1"
          stroke="#046b00"
          strokeLinecap="round"
          strokeWidth="2"
          transform="translate(1314.5 347.5)"
          x2="10.25"
          y2="9.225"
        />
        <line
          data-name="Line 2"
          fill="none"
          id="Line_2"
          stroke="#046b00"
          strokeLinecap="round"
          strokeWidth="2"
          transform="translate(1314.5 347.5)"
          x1="10.25"
          y2="9.225"
        />
      </g>
    </g>
  </svg>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-colorPrimary"
    style={
      Object {
        "marginRight": "10px",
      }
    }
  >
    کاربری با این مشخصات یافت نشد!
  </p>
</div>
`;
